pages/linux/tcpkill.md

```diff
@@ -0,0 +1,7 @@
+# tcpkill
+
+> Kills specified in-progress TCP connections.
+
+- Kill in-progress connections at a specified interface, host and port:
+
+`tcpkill -i {{eth1}} host {{192.095.4.27}} and port {{2266}}`
```

Comment:
.095 ? I have never seen an IP like this.

Reply:
Yeah, didn't think leading zeros were a thing in IPs. 🤔